solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -109,4 +109,4 @@ Run IO Sanity on your CORTX Cluster to validate if you are able to create bucket
 
 Tested by:
 
-* May 06, 2022: Rahul Shenoy (rahul.shenoy@seagate.com) - Windows + VMware Workstation 16 + CentOS 7.9 Linux
\ No newline at end of file
+* May 06, 2022: Rahul Shenoy (rahul.shenoy@seagate.com) - Windows '+' VMware Workstation 16 '+' CentOS 7.9 Linux
```

Comment:
why these single quotes added for `+` sign? please remove it

Reply:
@gauravchaudhari02 Please find the codeacy error : https://app.codacy.com/gh/Seagate/cortx-re/file/71107647120/issues/source?bid=27005559&fileBranchId=27005559#l112

Reply:
what if we put `,` instead of `+` sign?

Reply:
We can. Let me try that as well.

Reply:
So we are good with the above changes. Hence resolving the conversation.